x-pack/legacy/plugins/canvas/public/components/workpad_header/workpad_export/index.ts

```diff
@@ -6,7 +6,7 @@
 
 import { connect } from 'react-redux';
 import { compose, withProps } from 'recompose';
-import * as jobCompletionNotifications from '../../../../../reporting/public/lib/job_completion_notifications';
+import { jobCompletionNotifications } from '../../../../../../../plugins/reporting/public';
```

Comment:
Just a thought: this looks like a loose end from https://github.com/elastic/kibana/pull/47283 and I'm wondering why the Reporting notifications need to be explicitly imported here. Maybe Canvas doesn't register with whatever global notifications that works for the "classical" applications?

Reply:
Yeah, this is an interesting one... I wanted to dive in deeper to this but didn't want to get too distracted from getting this to just work.

Reply:
@elastic/kibana-canvas folks _ping_

Reply:
I'm not sure why this is like this. I'll have to dig into what this should look like. Probably not a blocker for this PR, though, I would think